enterprise/api/v1/views/enterprise_group.py

```diff
@@ -157,18 +171,34 @@ def assign_learners(self, request, group_uuid):
             customer = group.enterprise_customer
         except models.EnterpriseGroup.DoesNotExist as exc:
             raise Http404 from exc
-        if requested_emails := request.POST.dict().get('learner_emails'):
+        import pdb
+        pdb.set_trace()
+        if requested_emails := request.POST.getlist('learner_emails'):
```

Comment:
using `request.POST.getlist` here because default return value is `None` if the key doesn't exist. If we use `request.data.getlist`, we get an error `AttributeError: 'dict' object has no attribute 'getlist'`